moveit_core/trajectory_processing/src/time_optimal_trajectory_generation.cpp

```diff
@@ -881,36 +881,10 @@ bool TimeOptimalTrajectoryGeneration::computeTimeStamps(robot_trajectory::RobotT
 
   // Validate scaling
   double velocity_scaling_factor = 1.0;
```

Comment:
As mentioned above, I think this default of 1.0 should just be moved into `verifyScalingFactor()` to make it simpler.

Reply:
Makes sense. I'll update